.github/templates/common.yml.j2

```
@@ -7,7 +7,8 @@
 
 # NOTE: If testing pytorch/builder changes you can change this variable to change what pytorch/builder reference
 #       the binary builds will check out
-{%- set builder_branch = "main" -%}
+# personal branch test
+{%- set builder_branch = "zhuhong61:zh/conda_abi" -%}
```

Can you please try something like that? Or you might need to change the repo address as well
```suggestion
{%- set builder_branch = " pull/1095/head" -%}n```